src/__experimental__/components/date-range/date-range.component.js

```diff
@@ -32,11 +41,21 @@ class DateRange extends React.Component {
     /** Props for the child start Date component */
     startDateProps: PropTypes.shape({ ...DateInput.propTypes, value: PropTypes.string }),
     /** Props for the child end Date component */
-    endDateProps: PropTypes.shape({ ...DateInput.propTypes, value: PropTypes.string })
+    endDateProps: PropTypes.shape({ ...DateInput.propTypes, value: PropTypes.string }),
+    name: PropTypes.string,
```

Comment:
Please can you add some formatted description comments.

Reply:
I assume you mean for `name` and `id`?